src/screens/Onboarding/Onboarding.stories.tsx

```diff
@@ -43,46 +60,237 @@ export const Default: Story = {
+    });
+    await userEvent.click(runTestsButton);
+  }),
+  render: (args) => {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const [gitInfo, setGitInfo] = React.useState(args.gitInfo);
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const [localBuildProgress, setLocalBuildProgress] = React.useState(args.localBuildProgress);
+    return (
+      <>
+        <button
+          type="button"
+          style={{ position: "absolute", right: 0, bottom: 0 }}
+          onClick={() => setGitInfo({ branch: "main", uncommittedHash: "changed-hash" })}
+        >
+          Change Git
+        </button>
+        <meta.component
+          {...args}
+          gitInfo={gitInfo}
+          startDevBuild={() =>
+            setLocalBuildProgress({
+              ...INITIAL_BUILD_PAYLOAD,
+              currentStep: "upload",
+              buildProgressPercentage: 30,
+            })
+          }
+          localBuildProgress={localBuildProgress}
+        />
+      </>
+    );
+  },
+  parameters: {
+    ...BaselineSaved.parameters,
+    ...withFigmaDesign(
+      "https://www.figma.com/file/GFEbCgCVDtbZhngULbw2gP/Visual-testing-in-Storybook?type=design&node-id=304-318481&t=3EAIRe8423CpOQWY-4"
+    ),
+  },
 };
 
 export const ChangesFound: Story = {
-  parameters: withFigmaDesign(
-    "https://www.figma.com/file/GFEbCgCVDtbZhngULbw2gP/Visual-testing-in-Storybook?type=design&node-id=352-258984&t=3EAIRe8423CpOQWY-4"
-  ),
+  ...RunningFirstTest,
+  render: (args) => {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const [gitInfo, setGitInfo] = React.useState(args.gitInfo);
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const [localBuildProgress, setLocalBuildProgress] = React.useState(args.localBuildProgress);
+    return (
+      <>
+        <button
+          type="button"
+          style={{ position: "absolute", right: 0, bottom: 0 }}
+          onClick={() => setGitInfo({ branch: "main", uncommittedHash: "changed-hash" })}
+        >
+          Change Git
+        </button>
+        <meta.component
+          {...args}
+          gitInfo={gitInfo}
+          startDevBuild={() =>
+            setLocalBuildProgress({
+              ...INITIAL_BUILD_PAYLOAD,
+              buildProgressPercentage: 100,
+              currentStep: "complete",
+            })
+          }
+          localBuildProgress={localBuildProgress}
+        />
+      </>
+    );
+  },
+  parameters: {
+    ...BaselineSaved.parameters,
+    ...withFigmaDesign(
+      "https://www.figma.com/file/GFEbCgCVDtbZhngULbw2gP/Visual-testing-in-Storybook?type=design&node-id=352-258984&t=3EAIRe8423CpOQWY-4"
+    ),
+  },
 };
 
-export const Done: Story = {
-  parameters: withFigmaDesign(
-    "https://www.figma.com/file/GFEbCgCVDtbZhngULbw2gP/Visual-testing-in-Storybook?type=design&node-id=304-318693&t=3EAIRe8423CpOQWY-4"
-  ),
+// TODO: This design for an error in the Onboarding is incomplete
```

Comment:
Just highlighting this in case it is meant to be part of this PR

Reply:
I will remove this story, as this story does technically run. There was no design for it, but if there was an error, it will display the screen given. If an improvement is necessary we will find that in design review.